src/app/modules/feature-flags/feature-flags.service.ts

```diff
@@ -23,3 +24,49 @@ export const getEnabledFlags = (): ResultAsync<string[], DatabaseError> => {
     okAsync(enabledFlagsDocs.map((doc) => doc.name)),
   )
 }
+
+/**
+ * Wrapper over getEnabledFlags function to gracefully handle errors
+ * and returns the `options.fallbackValue` instead.
+ *
+ * @example
+ * ```
+ * const logMeta = {
+ *  action: 'linkStripeAccountToForm',
+ * }
+ * return getFeatureFlag(featureFlags.validateStripeEmailDomain, {
+ *  fallbackValue: false,
+ *  logMeta,
+ * })
+ *.andThen((shouldValidateStripeEmailDomain) => ...)
+ *```
+ *
+ * @param flag
+ * @param options.fallbackValue the value to fall back to in the event that there's an error retrieving the feature flag
+ * @returns boolean that represents the status of the feature flag or the fallback value
+ */
+export const getFeatureFlag = (
```

Comment:
Can be in a separate PR, but we should test this!

Reply:
Or not, since we will be trying out feature flag platforms like LaunchDarkly / Growthbook 🤔

Reply:
Yeah i think it's better that we look towards those tools.

Reply:
Yea agreed! :)